csi/server/plugin/opensds/controller.go

```diff
@@ -451,30 +453,125 @@ func (p *Plugin) ControllerGetCapabilities(
 					},
 				},
 			},
+			&csi.ControllerServiceCapability{
+				Type: &csi.ControllerServiceCapability_Rpc{
+					Rpc: &csi.ControllerServiceCapability_RPC{
+						Type: csi.ControllerServiceCapability_RPC_CREATE_DELETE_SNAPSHOT,
+					},
+				},
+			},
```

Comment:
Need to add capablity for LIST_SNAPSHOTS too

Reply:
LIST_SNAPSHOTS has been added